examples/custom-node/src/primitives/extended_op_tx_envelope.rs

```diff
@@ -119,28 +120,24 @@ where
     }
 }
 
-impl<T> InMemorySize for ExtendedOpTxEnvelope<T>
+impl<B, T> InMemorySize for ExtendedTxEnvelope<B, T>
 where
+    B: InMemorySize,
     T: InMemorySize,
 {
     fn size(&self) -> usize {
         delegate!(self => tx.size())
     }
 }
 
-impl<T> SignedTransaction for ExtendedOpTxEnvelope<T>
+impl<T, B> SignedTransaction for ExtendedTxEnvelope<B, T>
```

Comment:
let's swap the order of generics here
```suggestion
impl<B, T> SignedTransaction for ExtendedTxEnvelope<B, T>
```